src/modules/commands/delete.command.ts

```diff
@@ -1,15 +1,10 @@
-import {CustomDomainFactory} from "../domains/custom/custom.domain.factory";
-import {Domain} from "../domains/domain";
 import {Service} from "../services/services.crud";
 
 export const deleteCommand = async (request, reply) => {
-    const domainObjectsFactory = new CustomDomainFactory(request.params.entity, request.query);
-    const entity : Domain = domainObjectsFactory.instantiate();
     const crudService = request.diScope.resolve('crudService') as Service
-    crudService.delete(entity);
+    crudService.delete(request.params.id);
```

Comment:
I would keep sending the entity, then we forget about strings, ints...

Reply:
Good idea.